update(doc): Add deprecated functions (#657)
scaleway/docs-content: compute/functions/reference-content/functions-lifecycle.mdx

## Available runtimes

### NodeJS
* Node 8 (node8): **Deprecated**,, you won't be able to create new function using this runtime starting from August, the 1st 2022

> * Node 10 (node10): **Deprecated**,, you won't be able to create new function using this runtime starting from August, the 1st 2022

Reviewer: ```suggestion
* Node 10 (node10): **Deprecated**, you will not be able to create new functions using this runtime starting from August 1st 2022
```

* Node 14 (node14)
* Node 16 (node16)

> * Node 17 (node17): Security support ended June 1st 2022, **deprecated**,, you won't be able to create new function using this runtime starting from July, the 1st 2022

Reviewer: ```suggestion
* Node 17 (node17): Security support ended June 1st 2022. **Deprecated**, you will not be able to create new functions using this runtime starting from July 1st 2022
```

* Node 18 (node18) (beta): latest LTS available

### Python

> * Generic Python 2 (python2): **Deprecated**, you won't be able to create new function using this runtime starting from August, the 1st 2022

Reviewer: ```suggestion
* Generic Python 2 (python2): **Deprecated**, you will not be able to create new functions using this runtime starting from August 1st 2022
```

> * Generic Python 3 (python3): is now Python 3.7, **deprecated** you won't be able to create new function using this runtime starting from August, the 1st 2022

Reviewer: ```suggestion
* Generic Python 3 (python3): Now Python 3.7. **Deprecated** you will not be able to create new functions using this runtime starting from August 1st 2022
```

* Python 3.7 (python37)
* Python 3.8 (python38)
* Python 3.9 (python39)
* Python 3.10 (python310)

### Go

> * Generic Go (golang): is now Go 1.13, **deprecated** you won't be able to create new function using this runtime starting from August, the 1st 2022

Reviewer: ```suggestion
* Generic Go (golang): Now Go 1.13. **Deprecated** you will not be able to create new functions using this runtime starting from August 1st 2022
```
